### PR TITLE
New post: Getting Started with Hugging Face Inference Endpoints

### PR DESCRIPTION
**Pull request by juliensimon**

### CODE REVIEW

File: inference-endpoints.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+Deploying models in production usually requires jumping through a series of hoops. Packaging your model in a container, provisioning the infrastructure, creating your prediction API, securing it, scaling it, monitoring it, and more. Let's face it: building all this plumbing takes valuable time away from doing actual machine learning work. Unfortunately, it can also go awfully wrong.
+
+We strive to fix this problem with the newly launched Hugging Face [Inference Endpoints.](https://huggingface.co/inference-endpoints). In the spirit of making machine learning ever simpler without compromising on state-of-the-art quality, we've built a service that lets you deploy Transformer models directly from the [Hugging Face hub](https://huggingface.co) to managed infrastructure on your favorite cloud in just a few clicks. Simple, secure, and scalable: you can have it all.
```
Comment (philschmid) [inline inference-endpoints.md:31]:
```suggestion
We strive to fix this problem with the newly launched Hugging Face [Inference Endpoints](https://huggingface.co/inference-endpoints). In the spirit of making machine learning ever simpler without compromising on state-of-the-art quality, we've built a service that lets you deploy machine learning models directly from the [Hugging Face hub](https://huggingface.co) to managed infrastructure on your favorite cloud in just a few clicks. Simple, secure, and scalable: you can have it all.
```

File: inference-endpoints.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+Let me show you how this works!
+
+### Deploying a model on Inference Endpoints
+
+
+Looking at the list of [tasks](https://huggingface.co/docs/inference-endpoints/supported_tasks) that Inference Endpoints support, I decide to deploy a Swin image classification model that I recently fine-tuned with [AutoTrain](https://huggingface.co/autotrain) on the [food101](https://huggingface.co/datasets/food101) dataset . If you're interested in how I built this model, this [video](https://youtu.be/uFxtl7QuUvo) will show you the whole process.
```
Comment (philschmid) [inline inference-endpoints.md:38]:
```suggestion
Looking at the list of [tasks](https://huggingface.co/docs/inference-endpoints/supported_tasks) that Inference Endpoints support, I decided to deploy a Swin image classification model that I recently fine-tuned with [AutoTrain](https://huggingface.co/autotrain) on the [food101](https://huggingface.co/datasets/food101) dataset. If you're interested in how I built this model, this [video](https://youtu.be/uFxtl7QuUvo) will show you the whole process.
```

File: inference-endpoints.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+Now, let's increase our security level and deploy a private endpoint.
+ 
+### Deploying a Private Inference Endpoint
+
+Repeating the steps above, I select `Private` this time.
+
+This opens a new box asking me for the identifier of the AWS account in which the endpoint will be visible. I enter the appropriate id and click on `Create Endpoint`. 
+
+Not sure about your AWS account id? Here's an AWS CLI one-liner for you: `aws sts get-caller-identity --query Account --output text`
```
Comment (philschmid) [inline inference-endpoints.md:141]: Nice 😍

**Review by philschmid — APPROVED** [review]
I love it 😍 it is super short and shows how easy it is to deploy private endpoints 🔥 That's awesome! I left two comments :)

File: inference-endpoints.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+[{"score":0.9998466968536377,     "label":"hummus"},
+ {"score":0.00006414744711946696, "label":"falafel"},
+ {"score":6.4065129663504194e-6,  "label":"escargots"},
+ {"score":5.819705165777123e-6,   "label":"deviled_eggs"},
+ {"score":5.532585873879725e-6,   "label":"shrimp_and_grits"}]
+```
+
+This is all there is to it. Once I'm done testing, I delete the endpoints that I've created to avoid unwanted charges. I also delete the VPC Endpoint in the AWS console.
+
+### Now it's your turn!
```
Comment (philschmid) [inline inference-endpoints.md:178]: @juliensimon maybe can add a section before `now its your turn!` with the video of `phamily`: https://www.youtube.com/watch?v=20C9X5OYO2Q
Who is also talking about private link? Might be nice to add some quotes about what they shared.

**Review by jeffboudier — APPROVED** [review]
Love it! Super clear, end to end, thanks! 💪

File: inference-endpoints.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
+---
+title: Getting Started with Hugging Face Inference Endpoints
+thumbnail: /blog/assets/108_inference_endpoints/endpoints05.png
+---
```
Comment (osanseviero) [inline inference-endpoints.md:4]: Would it be possible to move the non-thumbnail assets to https://huggingface.co/datasets/huggingface/documentation-images/tree/main/blog please?

File: inference-endpoints.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+  <img src="assets/108_inference_endpoints/endpoints03.png">
+</kbd>
+
+After a few minutes, the endpoint is up and running, and its URL is visible.
+
+<kbd>
+  <img src="assets/108_inference_endpoints/endpoints04.png">
+</kbd>
+
+I can immediately test it by uploading an [image](assets/108_inference_endpoints/food.jpg) in the inference widget.
```
Comment (osanseviero) [inline inference-endpoints.md:81]: This image is 1.5MB, which will make it slow/expensive to read the blog post for some audiences

File: inference-endpoints.md [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+
+Of course, I can also invoke the endpoint directly with a few lines of Python code.
+
+```
+import requests, json
+
+API_URL = "https://oncm9ojdmjwesag2.eu-west-1.aws.endpoints.huggingface.cloud"
+
+headers = {
+  "Authorization": "Bearer MY_API_TOKEN",
```
Comment (osanseviero) [inline inference-endpoints.md:95]: Maybe add a docstring above this mentioning where they can find their token

File: inference-endpoints.md [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+5c7fbb4485cd8w7 2022-10-10T09:21:44.114Z 2022-10-10 09:21:44,114 | ERROR | Content type "None" not supported. Supported content types are: application/json, text/csv, text/plain, image/png, image/jpeg, image/jpg, image/tiff, image/bmp, image/gif, image/webp, image/x-image, audio/x-flac, audio/flac, audio/mpeg, audio/wave, audio/wav, audio/x-wav, audio/ogg, audio/x-audio, audio/webm, audio/webm;codecs=opus
+```
+
+Now, let's increase our security level and deploy a private endpoint.
+ 
+### Deploying a Private Inference Endpoint
+
+Repeating the steps above, I select `Private` this time.
+
+This opens a new box asking me for the identifier of the AWS account in which the endpoint will be visible. I enter the appropriate id and click on `Create Endpoint`.
```
Comment (osanseviero) [inline inference-endpoints.md:139]:
```suggestion
This opens a new box asking me for the identifier of the AWS account in which the endpoint will be visible. I enter the appropriate ID and click on `Create Endpoint`. 
```

**Review by osanseviero — APPROVED** [review]
Nice blog post!

**Comment by juliensimon** [comment]
Thanks everybody, I'll try to finalize it before the end of the week.

File: inference-endpoints.md (hunk 6 above)
Comment (juliensimon) [reply inference-endpoints.md:81]: I shrunk the image.

File: inference-endpoints.md (hunk 7 above)
Comment (juliensimon) [reply inference-endpoints.md:95]: ok.

File: inference-endpoints.md (hunk 4 above)
Comment (juliensimon) [reply inference-endpoints.md:180]: ok

**Pull request merged**